ci-operator/step-registry/mirror-images/by-oc-image-mirror/mirror-images-by-oc-image-mirror-commands.sh

```diff
@@ -0,0 +1,110 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+trap 'CHILDREN=$(jobs -p); if test -n "${CHILDREN}"; then kill ${CHILDREN} && wait; fi' TERM
+
+new_pull_secret="${SHARED_DIR}/new_pull_secret"
+qe_image="/tmp/mirror-images-list.yaml"
```

Comment:
If you want to keep this list for debugging, you can write it into `${ARTIFACT_DIR}/mirror-images-list.yaml`, so you can see this list in every build log.